src/main/target/common_pre.h

```diff
@@ -195,7 +195,6 @@
 #define USE_ESC_SENSOR_INFO
 #define USE_CRSF_CMS_TELEMETRY
 #define USE_BOARD_INFO
-#define USE_SMART_FEEDFORWARD
```

Comment:
This should be done in a separate pull request, as smart feedforward has been released and users are likely to open tickets if it suddenly disappears: #7391